package.json

```diff
@@ -17,7 +17,7 @@
     "test:cover": "COVERAGE=true yarn test"
   },
   "devDependencies": {
-    "@centerforopenscience/ember-osf": "https://github.com/CenterForOpenScience/ember-osf#feature/reviews#2017-12-21",
+    "@centerforopenscience/ember-osf": "https://github.com/CenterForOpenScience/ember-osf#df7716240c43af4ef42a191985436279e6e510ec",
```

Comment:
Can you run:
`yarn use-ember-osf-next-interfaces`
to set this to next interfaces instead of a pinned commit?

Reply:
Sure. Well, it does not seem like `use-ember-osf-next-interfaces` script is defined here yet both develop and release/next-interfaces.